content/blog/scale-kubernetes-cluster-using-hpe-greenlake-terraform-provider.md

```diff
@@ -0,0 +1,246 @@
+```
+
+# Terraform ready to apply
+
+Terraform executes the actions proposed in the Terraform plan and updates the resources. Run the command: **terraform apply** and type **yes** when asked to **Enter a value**.
+
+```shellsession
+$ terraform apply
+
+hpegl_caas_cluster.test: Refreshing state... [id=a32fabb9-7c19-42d1-9a38-ebf122810c0a]
+
+Terraform used the selected providers to generate the following execution plan. Resource actions are indicated with the following symbols:
+  ~ update in-place
+
+Terraform will perform the following actions:
+
+  # hpegl_caas_cluster.test will be updated in-place
+  ~ resource "hpegl_caas_cluster" "test" {
+        id                    = "a32fabb9-7c19-42d1-9a38-ebf122810c0a"
+        name                  = "tf-test"
+        # (17 unchanged attributes hidden)
+
+      + worker_nodes {
+          + count                = 1
+          + machine_blueprint_id = "0ac21c99-2fdb-491d-a590-a5016690b80b"
+          + name                 = "test-node-pool"
+        }
+    }
+
+Plan: 0 to add, 1 to change, 0 to destroy.
+
+Do you want to perform these actions?
+  Terraform will perform the actions described above.
+  Only 'yes' will be accepted to approve.
+
+  Enter a value: yes
+
+hpegl_caas_cluster.test: Modifying... [id=a32fabb9-7c19-42d1-9a38-ebf122810c0a]
+hpegl_caas_cluster.test: Still modifying... [id=a32fabb9-7c19-42d1-9a38-ebf122810c0a, 10s elapsed]
+hpegl_caas_cluster.test: Still modifying... [id=a32fabb9-7c19-42d1-9a38-ebf122810c0a, 1m10s elapsed]
+hpegl_caas_cluster.test: Still modifying... [id=a32fabb9-7c19-42d1-9a38-ebf122810c0a, 3m10s elapsed]
+hpegl_caas_cluster.test: Still modifying... [id=a32fabb9-7c19-42d1-9a38-ebf122810c0a, 5m10s elapsed]
+hpegl_caas_cluster.test: Still modifying... [id=a32fabb9-7c19-42d1-9a38-ebf122810c0a, 7m10s elapsed]
+hpegl_caas_cluster.test: Still modifying... [id=a32fabb9-7c19-42d1-9a38-ebf122810c0a, 9m10s elapsed]
+hpegl_caas_cluster.test: Still modifying... [id=a32fabb9-7c19-42d1-9a38-ebf122810c0a, 11m10s elapsed]
+hpegl_caas_cluster.test: Still modifying... [id=a32fabb9-7c19-42d1-9a38-ebf122810c0a, 13m10s elapsed]
+hpegl_caas_cluster.test: Still modifying... [id=a32fabb9-7c19-42d1-9a38-ebf122810c0a, 15m10s elapsed]
+hpegl_caas_cluster.test: Still modifying... [id=a32fabb9-7c19-42d1-9a38-ebf122810c0a, 17m10s elapsed]
+hpegl_caas_cluster.test: Still modifying... [id=a32fabb9-7c19-42d1-9a38-ebf122810c0a, 19m10s elapsed]
+hpegl_caas_cluster.test: Modifications complete after 19m18s [id=a32fabb9-7c19-42d1-9a38-ebf122810c0a]
+
+Apply complete! Resources: 0 added, 1 changed, 0 destroyed.
+```
+
+From HPE GreenLake edge-to-cloud platform, launch the HPE GreenLake Central console. Navigate to **Dashboard -> Manage your Private Cloud -> Containers** and select the **tf-test** cluster created. You will see additional nodes with **Node Pool** name as "**test-node-pool**" being created successfully.
```

Comment:
Recommend adding "the" prior to HPE GreenLake edge-to-cloud platform so it reads:
From the HPE GreenLake edge-to-cloud platform, launch...